lib/data_update_scripts/20210203104631_add_single_resource_admin_role_to_users_with_tech_admin.rb

```diff
@@ -0,0 +1,10 @@
+module DataUpdateScripts
+  class AddSingleResourceAdminRoleToUsersWithTechAdmin
```

Comment:
The reason for this Data Script is because we initially wanted to give permissions to Tech Admins to access this page. We instead decided to give permissions to the SingleResourceAdmins with resource type DataUpdateScript. You can read more about the details on [this thread](https://github.com/forem/internalEngineering/issues/276#issuecomment-771658292).